Intersect.Client.Core/MonoGame/Input/MonoInput.cs

```diff
@@ -311,6 +317,11 @@ public override void Update(TimeSpan elapsed)
                 var deltaX = (int)(gamePadState.ThumbSticks.Right.X * elapsed.TotalSeconds * 1000);
                 var deltaY = (int)(-gamePadState.ThumbSticks.Right.Y * elapsed.TotalSeconds * 1000);
 
+                if (deltaX != 0 || deltaY != 0)
+                {
+                    GameInput.Current.CursorMovementDevice = InputDeviceType.Controller;
```

Comment:
Ostensibly `GameInput.Current` refers to `this`, it should just be setting `CursorMovementDevice`